### DISCUSSION TITLE
New features - Master/sub documents for example

### DISCUSSION THREAD

**Discussion by BeginnerOne** [CollaboraOnline/online#5399] [Ideas]
I was wondering if there is a list of future enhancements list for CODE? 
I would like to add Master/SubDocuments to it. 
We have a dog club that is utilizing Nextcloud/Collabora for supporting the Board and over thirty committees to manage the operations of the club.  I have created a LibreOffice Master document that incorporates a lot of sub documents for the Policies and Procedures manual.  All of the sub documents are stored in Nextcloud but to assemble the finished product, they all must be maintained locally as well as in Nextcloud.  
If this is the wrong place to put this request please forgive me.

**Comment by pedropintosilva** [comment]
Hi @BeginnerOne and sorry for the delayed response (I'm afraid we really don't use the discussions here on GitHub, we use instead https://forum.collaboraonline.com/ ) but I will convert this directly into enhancement request (it will be visible in the issues)
